server/db/mongo/controllers/users.js

```diff
@@ -4,14 +4,19 @@ import {then} from '../../../utils/invokers';
 import jwt from 'jsonwebtoken';
 import {SECRET_KEY} from '../../../../config/env';
 
+const generateToken = user => ({
+  user,
+  token: jwt.sign(user._id, SECRET_KEY)
+});
+
 export const create = R.pipe(
```

Comment:
Also, is there a way to make this create/signUp method and signIn method look similar?
Currently `create` method has `(_, args)` inside `R.pipe`, but  `signIn` — vice versa.